kfp/transform_workflows/hack/test_pipeline.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+from kfp_support.workflow_support.utils import PipelinesUtils
+
+
+# Upload a pipeline. If the pipeline exists delete it and create a new one.
+def upload_pipeline(utils: PipelinesUtils, pipeline_name: str, pipeline_package_path: str):
+    """
+    Upload a pipeline. If the pipeline exists delete it and redeploy.
+
+    :param utils: object of type PipelinesUtils
+    :param pipeline_name: Name of the pipeline to be shown in the UI.
+    :param pipeline_package_path: Name of the pipeline to be shown in the UI.
+    return: Server response object containing pipleine id and other information.
+    """
+    pipeline = utils.upload_pipeline(pipeline_package_path=pipeline_package_path, pipeline_name=pipeline_name)
+    if pipeline is None:
```

Comment:
I think the simpler thing is to try to get pipeline by name and if it exists delete it before uploading

Reply:
Fixed. Thanks